CHANGES.txt

```diff
@@ -52,6 +52,11 @@ Documentation changes
 Test suite
 ----------
 * Fix AttributeError on Python 3.9 due to usage of ``base64.decodestring`` in tests (#210)
+* Make optional dependencies optional for tests (#260)
```

Comment:
One thing I noticed in this PR (it very well may have already been pointed out, I can't remember), is that with the new news fragment system PRs are added to the changelog in order of lowest to highest PR number, which may differ from other ets changelogs.
Not saying anything needs to change, just an observation

Reply:
Ahh. Good point. Not anything we need to change though.